packages/logs/src/boot/startLogs.spec.ts

```diff
@@ -339,71 +339,78 @@ describe('logs', () => {
       ])
     })
   })
+  ;[
+    { status: StatusType.error, message: 'Reached max number of errors by minute: 1' },
+    { status: StatusType.warn, message: 'Reached max number of warns by minute: 1' },
+    { status: StatusType.info, message: 'Reached max number of infos by minute: 1' },
+    { status: StatusType.debug, message: 'Reached max number of debugs by minute: 1' },
+    { status: 'unknown' as StatusType, message: 'Reached max number of customs by minute: 1' },
+  ].forEach(({ status, message }) => {
+    describe(`${status} logs limitation`, () => {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏FMU, those tests would also pass with a single limiter for all event types.
Could we ensure that:

- if one status reach its limit, we can still send other statuses
- two different custom statuses are accounted by the same limit

Reply:
Sure